devices/siglis.js

```diff
@@ -0,0 +1,42 @@
+const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
+const tz = require('zigbee-herdsman-converters/converters/toZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+const e = exposes.presets;
+
+
+module.exports = [
+    {
+        zigbeeModel: ['zigfred uno'],
+        model: 'ZFU-1D-CH',
+        vendor: 'Siglis',
+        description: 'zigfred uno Smart In-Wall Switch',
```

Comment:
Could you change this to `Zigfred uno smart in-wall switch` (brings it in line with the other descriptions), somehow I don't have edit rights

Reply:
Is `zigfred uno smart in-wall switch` also acceptable?
Reason: We avoid spelling "zigfred uno" with capitals in our Corporate Identity.
Of course I can change it like you proposed, if desired anyway.  :)

Reply:
That is also fine!